rancher/charts
# [dev-v2.7 ] rancher-monitoring fix crd update/delete

## Issue: 

https://github.com/rancher/rancher/issues/45575
https://github.com/rancher/rancher/issues/45167

## Problem

rancher-monitoring crd update/delete will fail on the first try

## Solution

Add required permissions

## Testing


## QA Testing Considerations

Update (from one version to another) / delete rancher-monitoring crd should pass

### Regressions Considerations


## Backporting considerations
v2.7, v2.8, v2.9
